benchmarks.cc

```diff
@@ -214,34 +214,40 @@ void run_throughput_benchmark(char const* name, Type<Queue>) {
 }
 
 template<class... Args>
-void run_throughput_benchmark(char const* name, Type<BoostSpScAdapter<boost::lockfree::spsc_queue<Args...>>>) {
+void run_throughput_spsc_benchmark(char const* name, Type<BoostSpScAdapter<boost::lockfree::spsc_queue<Args...>>>) {
     using Queue = BoostSpScAdapter<boost::lockfree::spsc_queue<Args...>>;
     run_throughput_benchmark<Queue>(name, 1000000, 1, 1); // spsc_queue can only handle 1 producer and 1 consumer.
 }
 
+template<class Queue>
+void run_throughput_spsc_benchmark(char const* name, Type<Queue>) {
+    run_throughput_benchmark<Queue>(name, 1000000, 1, 1); // ReaderWriterQueue can only handle 1 producer and 1 consumer.
+}
+
 void run_throughput_benchmarks() {
     std::printf("---- Running throughput benchmarks (higher is better) ----\n");
 
     int constexpr CAPACITY = 65536;
 
-    run_throughput_benchmark("boost::lockfree::spsc_queue", Type<BoostSpScAdapter<boost::lockfree::spsc_queue<unsigned, boost::lockfree::capacity<CAPACITY>>>>{});
+    run_throughput_spsc_benchmark("boost::lockfree::spsc_queue", Type<BoostSpScAdapter<boost::lockfree::spsc_queue<unsigned, boost::lockfree::capacity<CAPACITY>>>>{});
     run_throughput_benchmark("boost::lockfree::queue", Type<BoostQueueAdapter<boost::lockfree::queue<unsigned, boost::lockfree::capacity<CAPACITY - 2>>>>{});
-
+    
```

Comment:
Set your editor to remove trailing whitespace on save please. (I should add editorconfig).